{"name": "Переход в раздел «История заказов»", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16", "trace": "self = <test_account_page.TestAccountPage object at 0x0000013101416720>, data_new3 = {'email': 'pjames@example.net', 'name': 'Daniel', 'password': 'P7zaOWlp_z'}\ndriver_double = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"6d9d5383-ecc3-42e9-948f-9dbcbc8a5ecc\")>\n\n    @allure.title(\"Переход в раздел «История заказов»\")\n    def test_go_to_orders(self, data_new3, driver_double):\n        basepage = BasePage(driver_double)\n        basepage.click_button_login_account()\n        login = LoginPage(driver_double)\n        login.enter_login_password(data_new3)\n>       basepage.wait_active_create_order_click_account()\n\ntests\\test_account_page.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages\\base_page.py:171: in wait_active_create_order_click_account\n    self.wait_create_order_active()\npages\\base_page.py:129: in wait_create_order_active\n    WebDriverWait(self.driver, 30).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"6d9d5383-ecc3-42e9-948f-9dbcbc8a5ecc\")>, method = <function element_to_be_clickable.<locals>._predicate at 0x000001310148C9A0>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       RemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nE       WebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nE       NoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\nE       dom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Нажать на кнопку 'Войти в аккаунт'", "status": "passed", "start": 1720288039109, "stop": 1720288039349}, {"name": "Заполнить логин и пароль и войти в аккаунт", "status": "passed", "steps": [{"name": "Заполнить поле \"Email\"", "status": "passed", "parameters": [{"name": "email", "value": "'pjames@example.net'"}], "start": 1720288039349, "stop": 1720288039394}, {"name": "Заполнить поле \"Пароль\"", "status": "passed", "parameters": [{"name": "password", "value": "'P7zaOWlp_z'"}], "start": 1720288039394, "stop": 1720288039406}, {"name": "Подождать, что кнопка Вход активна", "status": "passed", "start": 1720288039406, "stop": 1720288039418}, {"name": "Нажать на кнопку \"Войти\"", "status": "passed", "start": 1720288039418, "stop": 1720288039635}], "parameters": [{"name": "data_new3", "value": "{'email': 'pjames@example.net', 'password': 'P7zaOWlp_z', 'name': 'Daniel'}"}], "start": 1720288039349, "stop": 1720288039635}, {"name": "Ожидание что кнопка личный кабиент будет активна и нажать на кнопку", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\pages\\base_page.py\", line 171, in wait_active_create_order_click_account\n    self.wait_create_order_active()\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\pages\\base_page.py\", line 129, in wait_create_order_active\n    WebDriverWait(self.driver, 30).until(\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Ожидание что создание заказа будет активно", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nRemoteError@chrome://remote/content/shared/RemoteError.sys.mjs:8:8\nWebDriverError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:193:5\nNoSuchElementError@chrome://remote/content/shared/webdriver/Errors.sys.mjs:511:5\ndom.find/</<@chrome://remote/content/shared/DOM.sys.mjs:136:16\n\n", "trace": "  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\pages\\base_page.py\", line 129, in wait_create_order_active\n    WebDriverWait(self.driver, 30).until(\n  File \"G:\\Diplom_Roman_Zogol_8\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1720288039636, "stop": 1720288069666}], "start": 1720288039636, "stop": 1720288069671}], "parameters": [{"name": "driver_double", "value": "'firefox'"}], "start": 1720288039109, "stop": 1720288069672, "uuid": "dc65b0ab-cd7a-4e0f-9a5d-83d16db29e55", "historyId": "b779bf5ccc65385f58e18fa6b746f6d5", "testCaseId": "a3e05c91b2440c47c2864e62a833520c", "fullName": "tests.test_account_page.TestAccountPage#test_go_to_orders", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_account_page"}, {"name": "subSuite", "value": "TestAccountPage"}, {"name": "host", "value": "DESKTOP-4LHP98R"}, {"name": "thread", "value": "22428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_account_page"}]}